pkg/interface/chat/src/js/components/settings.js

```diff
@@ -43,22 +46,25 @@ export class SettingsScreen extends Component {
   renderDelete() {
     const { props, state } = this;
 
-    let titleText = "Delete Chat";
-    let descriptionText = "Permanently delete this chat.";
-    let buttonText = "-> Delete";
+    let chatOwner = (deSig(props.match.params.ship) === window.ship);
 
-    if (deSig(props.match.params.ship) !== window.ship) {
-      titleText = "Leave Chat"
-      descriptionText = "You will no longer have access to this chat."
-      buttonText = "-> Leave";
-    }
+    let deleteButtonClasses = (chatOwner) ? 'b--red2 red2 pointer' : 'b--grey3 grey3 c-default';
+    let leaveButtonClasses = (!chatOwner) ? "pointer" : "c-default";
 
     return (
-      <div className="w-50 fl pl2 mt3">
-        <p className="body-regular">{titleText}</p>
-        <p className="label-regular gray mb3">{descriptionText}</p>
-        <a onClick={this.deleteChat.bind(this)}
-          className="pointer btn-font underline nice-red">{buttonText}</a>
+      <div>
+      <div className={"w-100 fl mt3 " + ((chatOwner) ? 'o-30' : '')}>
```

Comment:
Why display this at all if it's not an available option?

Reply:
Insisted in the mockups and conversations after — I imagine you want people to see what the owner can do and infer how chatrooms in general work.